File: Javascripts/checking if a string starts with something.js
function areYouPlayingBanjo(name) {
  return name + (name[0].toLowerCase() == 'r' ? ' plays' : ' does not play') + " banjo";
}





function areYouPlayingBanjo(name) {
  return name + (/^r/i.test(name) ? " plays " : " does not play ") + "banjo";
}




function areYouPlayingBanjo(name) {
  return name.toLowerCase().startsWith('r') ? name + " plays banjo" : name + " does not play banjo"
}

File: python/checking if a string starts with something.py
def areYouPlayingBanjo(name):
    if name[0].lower() == 'r':
        return name + ' plays banjo'
    else:
        return name + ' does not play banjo'





def areYouPlayingBanjos(name):
    return name + (' plays' if name[0].lower() == 'r' else ' does not play') + " banjo";